src/SSHLibrary/library.py

```diff
@@ -804,6 +804,7 @@ def login(self, username, password, delay='0.5 seconds'):
         return self._login(self.current.login, username, password, delay)
 
     def login_with_public_key(self, username, keyfile, password='',
+                              allow_agent=False, look_for_keys=False,
                               delay='0.5 seconds'):
```

Comment:
Adding new arguments before existing `delay` is backwards incompatible if someone has used this keyword like

```
Login with public key    ${user}    ${keys}    ${pwd}    10 seconds
```
because now that `10 seconds` would be passed in as `allow_agent`. I don't think this is a huge problem because you can pass delay as named argument like `delay=10 seconds` and I assume most users have used that approach. Such a change should, however, be mentioned in the release notes, and the related issue should get a `bwic` label so that we don't forget about it.

Reply:
Yeah, we should definetly mention it in the release notes. Added the `bwic` label to the issue.